evm/src/assets/TbrUser.sol

```diff
@@ -609,35 +611,12 @@ abstract contract TbrUser is TbrBase {
     return (abi.encodePacked(peer, baseFee, maxGasDropoff, paused), offset);
   }
 
-  receive() external payable {
-    // NOTE: all values retrieved here need to be immutables to avoid going over the 2300 gas stipend
-    if (!gasErc20TokenizationIsExplicit)
-      revert GasTokenNotSupported();
-
-    if (msg.sender != address(gasToken))
-      revert GasTokenOnlyAcceptedViaWithdrawal();
-  }
-}
-
-//TODO clean up and move this sort of functionality into the Solidity SDK _ASAP_
-library TokenBridgeVAAParser {
-  using BytesParsing for bytes;
-
-  uint private constant _TBR_V3_VAA_LENGTH_PREFIX_SIZE = 2;
-  uint private constant _VAA_SIGNATURE_ARRAY_OFFSET = 1 /*version*/ + 4 /*guardianSet*/;
-  uint private constant _VAA_SIGNATURE_SIZE = 1 /*guardianSetIndex*/ + 65 /*signaturesize*/;
-  uint private constant _VAA_EMITTER_CHAIN_SKIP = 4 /*timestamp*/ + 4 /*nonce*/;
-  uint private constant _VAA_TOKEN_AMOUNT_SKIP =
-    32 /*emitter address*/ + 8 /*sequence*/ + 1 /*consistencyLevel*/ + 1 /*payload id*/;
-  uint private constant _VAA_TOKEN_BRIDGE_RECIPIENT_SKIP =
-    32 /*token bridge recipient address (us)*/ + 2 /*token bridge recipient chain id (our chain)*/;
-
   function parse(
```

Comment:
Should we call this `parseRedeem` or something? I think it's a bit too vague to call it just `parse` now that it's not namespaced to a library.